src/main/kotlin/org/taymyr/lagom/elasticsearch/indices/dsl/CreateIndex.kt

```diff
@@ -17,7 +41,10 @@ class CreateIndex(
     )
 
     @Immutable
-    data class Mapping(val properties: Map<String, MappingProperty>)
+    data class Mapping @JvmOverloads constructor(
+        val properties: Map<String, MappingProperty>,
+        val dynamic: DynamicType? = DynamicType.DYNAMIC_TRUE
```

Comment:
I think that the default value should be `null`. (ES must use its default value)